Release_Process.md

```diff
@@ -0,0 +1,42 @@
+# Confidential Container Release Process
+
+## Process
+- Code Freeze
+- Issue created from [Release Check list template](https://github.com/confidential-containers/community/issues/new?assignees=&labels=&template=release-check-list.md&title=%5BRelease%5D+Check+list+for+v%3CTARGET_RELEASE%3E)
+- "[Release Owner](https://github.com/confidential-containers/community/wiki/Release-Team-Rota)" identifies owners for Check List Items
+- Check List Items completed
+- Release Complete
+- Removal of Code Freeze and release announced.
+
+Release Dates will be set mid-week, and no one should feel an expectation to work unsocial hours to meet a release date. Communication and expectation setting is important, and can be used to involve more people from the community to address any Issues blocking release. From the point of code freeze, the release is being built so the release date is an estimate. Releasing early or late along with ideas for improvements/changes for future releases will be reflected on after release.
+
+
+## Requirements to help with Release Process
+- Availability during the release period to complete Tasks 
+- **TODO** What access is needed/suggested?
+- GitHub permissions to push tags and create releases in CoCo repositories
+
+## Expectations on Release Owners
+The release owner is not expected to complete the Release Check List alone and the majority of tasks on the check list can be delegated. Delegation will help others become confident in assisting with the release and share the burden, but does rely on community members volunteering.
+
+The key responsibility which can not be delegated is communication and oversight
+- communicate and ensure code freeze with repository maintainers
+- ensure the checklist is completed
+- ensure the checklist is updated to reflect any new checklist items
+- report status to the community
+- announce the new release and end of code freeze.
+
+
+## Code Freeze - Expectations from the Community
+- During the Code Freeze period no code should be merged.
```

Comment:
Some changes are appropriate during the freeze period such as updating dependencies (by the release team). We might want to mention this. Maybe it would be good to have a short list of exemptions including documentation.

Reply:
Added a bit of clarification here